Clients/src/presentation/components/Table/RisksTable/index.tsx

```
@@ -130,31 +148,72 @@ const RiskTable: React.FC<RiskTableProps> = ({
+                    const displayCount = 1;
+                    const showMore = projects.length > displayCount;
+                    const displayed = projects.slice(0, displayCount).join(', ');
+                    const moreCount = projects.length - displayCount;
+                    return (
+                      <Tooltip
+                        title={
+                          <>
+                            {projects.map((title, idx) => (
+                              <div key={idx}>{title}</div>
+                            ))}
+                          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)